lib/BloqadeQMC/src/ising/Rydberg.jl

```diff
@@ -1,23 +1,23 @@
 using Base.Iterators
 
-
 abstract type AbstractRydberg{O <: AbstractOperatorSampler} <: AbstractLTFIM{O} end
```

Comment:
what is this abstract type for? if you have a `Rydberg` object already.

Reply:
We're taking care of all the typing in a separate PR :)